src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -10790,54 +11042,88 @@ void emitter::emitDispVectorReg(regNumber reg, insOpts opt, bool addComma)
         printf(", ");
 }
 
-/*****************************************************************************
- *
- *  Display an vector register index suffix
- */
+//------------------------------------------------------------------------
+// emitDispVectorRegIndex: Display a SIMD vector register name with element index
```

Comment:
Would this be clearer named `emitDispVectorRegWithIndex`? or `emitDispVectorRegWithSizeAndIndex`

Reply:
The "official" name for this is **SIMD vector element name** (see *C1.2.5 Register names*) so we might also rename it to `emitDispVectorElementName`